### ISSUE TITLE
"Only absolute URLs are supported" error for publishing the Bot

### ISSUE THREAD

**Issue by pankajsurti** [microsoft/BotFramework-Composer#3581]
## Browser

- [x ] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [ x] Windows
- [ ] Ubuntu

## To Reproduce

Publish to Selected Profile
Select the profile. Which worked fine earlier when the Bot was created first time with LUIS and QnA

![image](https://user-images.githubusercontent.com/108044/86801435-9beeaa80-c039-11ea-9432-7a6e18ad7e4d.png)

## Additional context
![image](https://user-images.githubusercontent.com/108044/86802162-57174380-c03a-11ea-922d-370f6ec6f16e.png)

**Comment by cwhitten** [comment]
hi @pankajsurti - the team has recently addressed this and it will be included in an upcoming release (1.0.2)

**Comment by pankajsurti** [comment]
> hi @pankajsurti - the team has recently addressed this and it will be included in an upcoming release (1.0.2)

@cwhitten  When will be 1.0.2 released?

**Comment by cwhitten** [comment]
@pankajsurti 1.0.2 has been released.

**Comment by briandenicola** [comment]
Has this been addressed in 1.0.2?  I am getting the same error while publishing an existing bot.

My publish setting file looks like:
```
{
  "accessToken": "",
  "hostname": "bjdbot001.azurewebsites.net",
  "settings": {
    "MicrosoftAppId": "64d3ce26-ccba-47d2-98b6-*************",
    "MicrosoftAppPassword": "RVNE2ZVk26Bc!******************"
  }
}
```
I've tried bjdbot001, https://bjdbot01.azurewebsites.net, https://bjdbot01.scm.azurewebsites.net, and bjdbot01.scm.azurewebsites.net as well. I always get the same error. 

![image](https://user-images.githubusercontent.com/1042537/90336525-07fdf000-dfa2-11ea-855c-771d16562476.png)
![image](https://user-images.githubusercontent.com/1042537/90336507-e8ff5e00-dfa1-11ea-982a-85a274b79909.png)
